src/app/components/shared/datetime-formats/datetime/zoned-datetime/zoned-datetime.component.ts

```diff
@@ -18,10 +19,16 @@ export class ZonedDateTimeComponent extends AbstractDatetimeComponent {
     super();
   }
 
+  // if we set the timezone to null or undefined, we want to remove the explicit timezone
+  // by setting the explicit timezone to the implicit timezone
```

Comment:
that's not an accurate comment is it? because the code sets the explicit tz to null, not to the implicit
```suggestion
  // which will result in any implicit time zone taking effect
```